be/src/exec/iceberg/iceberg_delete_builder.cpp

```diff
@@ -52,6 +59,12 @@ Status ParquetPositionDeleteBuilder::build(const std::string& timezone, const st
             }
         }
     }
+
+    // eof is expected, otherwise propagate error
+    if (!status.is_end_of_file()) {
+        LOG(WARNING) << status;
```

Comment:
The warning message is some simple, how could the users know the problem?

Reply:
this is to print warning log to ease debugging, not for user